cryptodoc/src/06_pubkey_key_generation.rst

```diff
@@ -477,3 +477,140 @@ in constant time on identical hardware if the hash function computations
+   | :ref:`Modern Variant <pubkey_key_generation/dilithium/modes>`                     | :srcref:`src/lib/pubkey/dilithium/dilithium`                                         | "Modern" instantiations of primitives                                                                                                                                                  |
+   +-----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | :ref:`AES Variant <pubkey_key_generation/dilithium/modes>`                        | :srcref:`src/lib/pubkey/dilithium/dilithium_aes`                                     | "AES" instantiations of primitives                                                                                                                                                     |
+   +-----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | :ref:`Polynomial Operations <pubkey_key_generation/dilithium/polynomials>`        | :srcref:`src/lib/pubkey/dilithium/dilithium_common/dilithium_polynomials.h`          | Polynomials and operations on them                                                                                                                                                     |
+   +-----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+   | Dilithium                                                                         | :srcref:`src/lib/pubkey/dilithium/dilithium_common/dilithium.h`                      | Dilithium :ref:`Keys <pubkey_key_generation/dilithium/keys>`, :ref:`Signature Creation <pubkey_signature/dilithium/sig>`, :ref:`Signature Validation <pubkey_signature/dilithium/val>` |
+   +-----------------------------------------------------------------------------------+--------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+.. _pubkey_key_generation/dilithium/modes:
+
+**Modes, Constants and Symmetric Primitives**
+
```

Comment:
I don't remember either. But I noticed we do the same in Kyber but not SPHINCS+. I treated these headers more as LaTeX paragraphs but I'm fine with treating them more like "sections". These will however appear in the Table of Contents (I noticed it's already cluttered for SPHINCS+).

So if we think it's too cluttered I'd say bold letters, otherwise I'd say using the RST headers. What do you think?

Reply:
I didn't notice we did the same for Kyber. I'm okay with the bold letter variant as it is.